symphony-bdk-examples/bdk-core-examples/build.gradle

```diff
@@ -16,5 +17,7 @@ dependencies {
 
     testCompileOnly 'org.projectlombok:lombok'
     testAnnotationProcessor 'org.projectlombok:lombok'
+
+    implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
```

Comment:
Should be transitive through `symphony-bdk-http-jersey2`. Can we do that @symphony-youri?

Reply:
To me we should do 2 things here:
- implementation 'org.glassfish.jersey.core:jersey-client' in the example module because in HttpClientExampleMain we import the generic type
- in bdk-core change implementation 'org.glassfish.jersey.core:jersey-client' to api and add the plugin java-library since we are leaking the GenericType in the HttpClient class